README.md

```diff
@@ -32,7 +32,7 @@ Install the JavaScript with:
 expo install react-native-svg
 ```
 
-📚 See the [**Expo docs**](https://docs.expo.io/versions/latest/sdk/svg/) for more info or jump ahead to [Usage](#Usage).
+📚 See the [**Expo docs**](https://docs.expo.io/versions/latest/sdk/svg/) for more info or jump ahead to [Usage](./USAGE.md).
```

Comment:
I think what we wanted here is the link to `Usage` part of this page, so it should be just `#usage`. I'll change it in your PR and it should work. (see `https://github.com/software-mansion/react-native-svg#usage`)